runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/extensions/custom/CustomRenderer.js

```diff
@@ -300,7 +307,7 @@ export default class CustomRenderer extends BaseRenderer {
             }
             return {
                 title: businessObject.label,
-                translateKey: translateKey,
+                actionId: actionId,
```

Comment:
no need to double variable, JS will substitute value for the field like variable name

Reply:
done